packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/summary/telephony-line-phone-order-summary.html

```diff
@@ -0,0 +1,210 @@
+            ></p>
+            <p
+                class="mb-1"
+                data-ng-bind="$ctrl.order.contact.address.line1"
+            ></p>
+            <p
+                class="mb-1"
+                data-ng-bind="$ctrl.order.contact.address.zip + ' ' + $ctrl.order.contact.address.city"
+            ></p>
+        </div>
+
+        <!-- SHIPPING MODE -->
+        <h3
+            class="mt-4"
+            data-translate="telephony_line_phone_order_shipping_mode"
+        ></h3>
+        <p
+            data-ng-if="$ctrl.order.shipping.mode === 'mondialRelay'"
+            data-translate="telephony_line_phone_order_shipping_mode_mondial_relay"
+            data-translate-values="{
+               name: $ctrl.order.shipping.relay.name,
+               street: $ctrl.order.shipping.relay.address,
+               city: $ctrl.order.shipping.relay.city
+           }"
+        ></p>
+        <p
+            data-ng-if="$ctrl.order.shipping.mode !== 'mondialRelay'"
+            data-translate="telephony_line_phone_order_shipping_mode_other"
+        ></p>
+
+        <!-- CONTRACTS -->
+        <h3
+            class="mt-4"
+            data-translate="telephony_line_phone_order_contracts"
+        ></h3>
+        <div
+            class="mb-4"
+            data-ovh-contracts="$ctrl.order.summary.contracts"
+            data-full-text="false"
+            data-ovh-contracts-validated="$ctrl.order.isContractsAccepted"
+        ></div>
+
+        <div data-ng-if="!$ctrl.phone">
+            <hr />
+            <label class="d-block mb-2">
+                <input
+                    type="radio"
+                    data-ng-value="true"
+                    data-ng-model="$ctrl.order.retract"
+                />
+                <span
+                    class="ml-2"
+                    data-translate="telephony_line_phone_order_retract"
+                >
+                </span>
+            </label>
+            <label class="d-block mb-2">
+                <input
+                    type="radio"
+                    data-ng-value="false"
+                    data-ng-model="$ctrl.order.retract"
+                />
+                <span
+                    class="ml-2"
+                    data-translate="telephony_line_phone_order_noretract"
+                >
+                </span>
+            </label>
+        </div>
+
+        <br />
+        <hr class="oui-horizontal-line_dark" />
+
+        <div>
+            <button
+                type="button"
+                class="oui-button oui-button_secondary oui-button_icon-left mt-2"
+                data-ng-click="$ctrl.backShipping()"
+            >
+                <span
+                    data-translate="telephony_line_phone_order_order_prev"
+                ></span>
+            </button>
+            <button
+                type="button"
+                class="oui-button oui-button_primary oui-button_icon-right float-right mt-2"
```

Comment:
```suggestion
                class="oui-button oui-button_primary oui-button_icon-right mt-2"
```
use `flex` if needed instead, as stated above